common/culture/innovations/00_early_modern_innovations.txt

```diff
@@ -0,0 +1,220 @@
+﻿#### Local Types ####
+@administration_1 = "gfx/interface/icons/culture_innovations/innovation_administration_01.dds"
+@administration_2 = "gfx/interface/icons/culture_innovations/innovation_administration_02.dds"
+@administration_3 = "gfx/interface/icons/culture_innovations/innovation_administration_03.dds"
+@civil_construction_1 = "gfx/interface/icons/culture_innovations/innovation_civil_construction_01.dds"
+@civil_construction_2 = "gfx/interface/icons/culture_innovations/innovation_civil_construction_02.dds"
+@leadership_1 = "gfx/interface/icons/culture_innovations/innovation_leadership_01.dds"
+@leadership_2 = "gfx/interface/icons/culture_innovations/innovation_leadership_02.dds"
+@raised_banner = "gfx/interface/icons/culture_innovations/innovation_raised_banner.dds"
+@fortifications = "gfx/interface/icons/culture_innovations/innovation_fortifications.dds"
+@siege_weapons = "gfx/interface/icons/culture_innovations/innovation_siege_weapons.dds"
+@levy_building = "gfx/interface/icons/culture_innovations/innovation_levy_building.dds"
+@maa_01 = "gfx/interface/icons/culture_innovations/innovation_maa_01.dds"
+@maa_02 = "gfx/interface/icons/culture_innovations/innovation_maa_02.dds"
+@weapons_and_armor_01 = "gfx/interface/icons/culture_innovations/innovation_weapons_and_armor_01.dds"
+@weapons_and_armor_02 = "gfx/interface/icons/culture_innovations/innovation_weapons_and_armor_02.dds"
+@knight = "gfx/interface/icons/culture_innovations/innovation_knight.dds"
+@hird = "gfx/interface/icons/culture_innovations/innovation_hird.dds"
+@majesty_01 = "gfx/interface/icons/culture_innovations/innovation_majesty_01.dds"
+@majesty_02 = "gfx/interface/icons/culture_innovations/innovation_majesty_02.dds"
+@majesty_03 = "gfx/interface/icons/culture_innovations/innovation_majesty_03.dds"
+@nobility_01 = "gfx/interface/icons/culture_innovations/innovation_nobility_01.dds"
+@nobility_02 = "gfx/interface/icons/culture_innovations/innovation_nobility_02.dds"
+@nobility_03 = "gfx/interface/icons/culture_innovations/innovation_nobility_03.dds"
+@nobility_04 = "gfx/interface/icons/culture_innovations/innovation_nobility_04.dds"
+@misc_inventions = "gfx/interface/icons/culture_innovations/innovation_misc_inventions.dds"
+
+@guns = "gfx/interface/icons/culture_innovations/innovation_gunpowder.dds"
+@production = "gfx/interface/icons/culture_innovations/innovation_production_line.dds"
+@trade = "gfx/interface/icons/culture_innovations/innovation_trade_company.dds"
+@tram = "gfx/interface/icons/culture_innovations/innovation_trams.dds"
+
+#culture_group_military
+innovation_better_seige = {
```

Comment:
If you check the vanilla you can see Paradox tried (and sometimes failed) to follow some pattern of how techs are placed.
So that's military.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/122899437-e0fc1180-d35c-11eb-95f5-ea2533beeb70.png)

</details>

Civic techs are a mess, but I would stick to the pattern of the last eras.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/122900040-67b0ee80-d35d-11eb-8071-7578e43b9f4c.png)

</details>

But anyway, I would place our techs in the same order.

Reply:
Tried to copy this to the best of my ability (some techs are different though in general)